polkadot/primitives/src/v5/mod.rs

```diff
@@ -109,6 +109,10 @@ pub trait TypeIndex {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Hash))]
 pub struct ValidatorIndex(pub u32);
 
+/// Type alias for chunk index. The chunk index held by a validator may not always be equal to its
+/// `ValidatorIndex`, so we use a separate type to make code easier to read.
+pub type ChunkIndex = ValidatorIndex;
```

Comment:
we should also use it in av-store. although I would go for a stronger typed new-type wrapper

Reply:
agree, I'll use a newtype wrapper here. granted that it's not used in the runtime, do I need to add it to vstaging primitives?